canonical/prometheus-k8s-operator
# Fix `stripPrefix`

## Issue
The fix in #512 removed `stripPrefix` as a workaround, but we still want the stripPrefix functionality.


## Solution
Revert #512 and use `web.route-prefix`.
- Fixes #511.
- Addresses #516.

Drive-by fixes:
- Fix code ordering issue that resulted in blocked status for missing web config file.
- Document manual tests.

## Testing Instructions
1. Relate `catalogue - prometheus - traefik` and test endpoint with browser. Make sure query results are non empty.
2. Repeat with TLS.

In tandem with:
- https://github.com/canonical/traefik-k8s-operator/pull/232
